tests/e2e/performance/multi_snapshots.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import time
+import datetime
+import yaml
+import tempfile
+import logging
+import subprocess
+
+from ocs_ci.ocs import constants
+
+ERRMSG = 'Error in command'
+
+log = logging.getLogger(__name__)
+
+# Dictionary to hold all test parameters that need to passed as environment
+# variables.
+# After all variables are read an validate, some more variables can be added
+# to this dictionary.
+params = {'KUBECONFIG': None, 'SNAPNUM': None, 'LOGPATH': None,
+          'FILESIZE': None, 'NSPACE': None, 'INTERFACE': None,
+          'PODNAME': None, 'PVCNAME': None}
+
+# Dictionary to hold the names of pods which holding logs of creation time
+log_names = {'start': None, 'end': []}
+
+snap_yaml = None  # define this parameter as global
+format = '%H:%M:%S.%f'
+
+log_file_name = os.path.basename(__file__).replace('.py', '.log')
+
+
+def msg_logging(msg):
+    """
+    This function is logging the message to the log file, and also print it
+    for the caller script output
+
+    Args:
+        msg (str): The message to log as info and print on the console
+
+    """
+    print(msg)
```

Comment:
what is the need to print a message to the console?

Reply:
with out that, output will not seen in the main console log